kubetools/kubernetes/api.py

```diff
@@ -41,9 +41,29 @@ def _get_k8s_apps_api(env):
     return client.AppsV1Api(api_client=api_client)
 
 
-def _get_k8s_batch_api(env):
+def check_if_cronjob_compatible(env):
+    settings = get_settings()
+
+    api_core = _get_k8s_core_api(env)
+    list_node = api_core.list_node().items
+    first_node_info = list_node[0].status.node_info
+    kubelet_version = first_node_info.kubelet_version.replace('v', '').split(".")
+    major_version = int(kubelet_version[0])
+    minor_version = int(kubelet_version[1])
```

Comment:
Parsing should be extracted in a function, and possibly use [an external library](https://pypi.org/project/version/) instead of reinventing the wheel.